messaging/src/android/java/com/google/firebase/messaging/cpp/RegistrationIntentService.java

```diff
@@ -37,11 +36,20 @@ public RegistrationIntentService() {
     super(TAG);
   }
 
+  /**
+   * Convenience wrapper over enqueueWork to either directly start the service (when running on
+   * pre-O platforms) or enqueue work for it as a job (when running on Android O and later).
+   */
+  public static void enqueueWork(Context context, Intent intent) {
```

Comment:
Is this getting called from somewhere else?

Reply:
This was included by mistake. I've removed it